app/src/main/java/za/org/grassroot2/presenter/activity/CreateActionPresenter.kt

```diff
@@ -114,10 +139,24 @@ constructor(private val networkService: NetworkService, private val dbService: D
         (task as Meeting).locationDescription = location
     }
 
-    fun setSubject(subject: String) {
+    fun setMeetingSubject(subject: String) {
         (task as Meeting).setSubject(subject)
     }
 
+    fun setTodoSubject(subject: String) {
+        (task as Todo).setSubject(subject)
+    }
```

Comment:
You could also add a setSubject method to Task interface and then just have one method